src/SummaryRow.tsx

```diff
@@ -21,20 +22,30 @@ function SummaryRow<R, SR>({
   bottom,
   'aria-rowindex': ariaRowIndex
 }: SummaryRowProps<R, SR>) {
+  let colSpan: number| undefined;
```

Comment:
```suggestion
  let colSpan: number | undefined;
```

Reply:
Changed the logic to use cells array and removed `colSpan` variable. It is a tiny bit performant as we do not loop all the columns